src/main/scala/io/iohk/ethereum/vm/BlockchainConfigForEvm.scala

```diff
@@ -35,13 +36,15 @@ case class BlockchainConfigForEvm(
     aghartaBlockNumber: BigInt,
     petersburgBlockNumber: BigInt,
     phoenixBlockNumber: BigInt,
+    magnetoBlockNumber: BigInt,
     chainId: Byte
 ) {
   def etcForkForBlockNumber(blockNumber: BigInt): EtcFork = blockNumber match {
     case _ if blockNumber < atlantisBlockNumber => BeforeAtlantis
     case _ if blockNumber < aghartaBlockNumber  => Atlantis
     case _ if blockNumber < phoenixBlockNumber  => Agharta
-    case _ if blockNumber >= phoenixBlockNumber => Phoenix
+    case _ if blockNumber < magnetoBlockNumber  => Phoenix
+    case _ if blockNumber >= magnetoBlockNumber => Magneto
```

Comment:
do you mind adding a task for creating the same mapping for eth berlin?

Reply:
It is done